API/Controllers/ReviewsController.cs

```diff
@@ -52,6 +52,19 @@ public async Task<List<Review>> GetAllReviews()
             return responseList;
         }
 
+        /*
```

Comment:
This usually isnt a huge deal when it comes to small projects but for clean code we like to delete commented out code. If this code is going to be used in the future for a feature of some sort, then you should put a TODO: "leaving because of blah" just for other developers knowledge.